examples/lang/erc20/src/lib.rs

```diff
@@ -130,7 +129,7 @@ contract! {
         }
 
         /// Transfers token from a specified AccountId to another AccountId.
-        fn transfer_impl(&mut self, env: &mut ink_model::EnvHandler, from: AccountId, to: AccountId, value: Balance) -> bool {
+        fn transfer_impl(&mut self, env: &mut EnvHandler<ink_core::env::ContractEnv<NodeRuntimeTypes>>, from: AccountId, to: AccountId, value: Balance) -> bool {
```

Comment:
Hmm this is really ugly but also a great motivation to hide this ugliness through the macros.

Reply:
... in a later PR

Reply:
Yes definitely! Remember earlier I had a type alias to avoid this but we decided to be explicit here for clarity.

Reply:
Clarity definitely wins over ugliness.